docs/development.md

```diff
@@ -176,3 +177,167 @@ $ yarn run [name_command]
 | `devrun`             | Cleans, transpiles, runs and watches the webpack-dev-server using `DEV` environment. |
 | `webpack-dev-server` | Runs the webpack-dev-server in port `9111`. |
 | `test`               | Runs the client side tests built using Jest.  |
+
+## 7. A/B Test framework
+
+In this section we will go through the steps and considerations that you must have when you want to set up a new test.
+
+### 7.1 API
+
+The AB test framework has the following methods:
+
+#### `init()`
+This function initialize the AB test framework by executing the following steps:
+1. Read the MVT (multi variant) id from the cookie of the user. If the user has no mvt is his or her browser, 
+it generate a new MVTid and save it in the cookie.
+
+2. From the MVT, it generate the `Participation` object. The steps to build that object are:
+   * Read the participation from `localStorage`
+   * Check if the test is active.
+   * if the test is not in `localStorage`, it assign one variant from the `mvtId`.
+   * Finally it override the variant if the user pass the variant in the url in the way of: `#ab-[name_of_test]=[variant_name]`.
+
+3. Save the `Participation` object in `localStorage`. 
+     
+
+#### `getVariantsAsString(participation: Participations)`
+It receive the participation object and returns everything as string in the following format:
```

Comment:
As a way of documenting the AB test framework file, first I thought it would be a good idea to start explaining the methods that the module exports. Since this method is being exported by the module, I had to include it here.